frontend/src/core/locale/getPluralExamples.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+
+import type { Locale } from './actions';
+
+
+export default function getPluralExamples(locale: Locale) {
+    const pluralsCount = locale.cldrPlurals.length;
+    const examples = {};
+
+    if (pluralsCount === 2) {
+        examples[locale.cldrPlurals[0]] = 1;
+        examples[locale.cldrPlurals[1]] = 2;
+    }
+    else {
+        // This variable is used in the pluralRule we eval in the while block.
+        let n = 0;
+        while (Object.keys(examples).length < pluralsCount) {
+            // This `eval` is not so much evil. The pluralRule we parse
+            // comes from our database and is not a user input.
+            // eslint-disable-next-line
+            const rule = eval(locale.pluralRule);
```

Comment:
Sorry, but I can't get this to work. I'm getting "Cannot get locale.cldrPlurals[pluralRule] because new Function is not an array index". I though `pluralRule()` would help.

Note that this particular use of `eval()` has been approved during two of our security audits (bug 1338247 and when defining the CSP policy). I'm curious how is `new Function()` more secure than `eval()`.

Reply:
Yeah, you need `const rule = pluralRule(n)`, too.

Reply:
Re the sec impact, https://stackoverflow.com/questions/4599857/are-eval-and-new-function-the-same-thing has good comments. It's not cool, but it's less hot. ;-)

Reply:
Nope, still won't work ("no arguments expected by new Function()").

And yes, I've fixed `locale.PluralRule`.

Reply:
Actually this works now - the "no arguments expected" is a Flow error. I can't figure out how to bypass it beyond adding a series of `// $FLOW_IGNORE` lines.

Reply:
Let's keep the eval then.